plugin.yml

```diff
@@ -9,6 +9,8 @@ configuration:
       type: string
     job-uuid-file-pattern:
       type: string
+    failure-format:
+      type: string
```

Comment:
Even tho it's not validated at runtime at the moment, let's add [an enum](https://spacetelescope.github.io/understanding-json-schema/reference/generic.html#enumerated-values) to limit it to the valid values:

```yml
failure-format:
  type: string
  enum:
    - classname
    - file
```